index.html

```diff
@@ -11,151 +11,236 @@
-            <li><a href="#"><img src="image/Vector-(2).png" alt="Social media"></a></li>
-            <li><a href="#"><img src="image/Vector-(3).png" alt="Social media"></a></li>
-          </ul>
-          <button type="button" class="button my-resume">Get my resume</button>
+      <section class="about">
+        <div class="about-section">
+          <div class="myself">
+            <h2>About<br>Myself</h2>
+            <p class="about-detail">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+            <h3><a href="" class="info-link">LET'S CONNECT</a></h3>
+            <ul class="about-social">
+              <li><a href="#"><img src="image/Vector-(4).png" alt="Email icon"></a></li>
+              <li><a href="#"><img src="image/Linkedin-icon.png" alt="Social media"></a></li>
+              <li><a href="#"><img src="image/Vector-(1).png" alt="Email icon"></a></li>
+              <li><a href="#"><img src="image/Vector-(2).png" alt="Social media"></a></li>
+              <li><a href="#"><img src="image/Vector-(3).png" alt="Social media"></a></li>
+            </ul>
+            <button type="button" class="button my-resume">Get my resume</button>
+          </div>
+          <div class="language-detail">
+            <div class="language-detail">
+              <ul>
+                <li class="lang icon">
+                  <span class="word">Language</span>
+                  <img src="image/arrow2.png" alt="dropdown arrow icon">          
+                </li>
+              </ul>
+            </div>
+            <div class="language-detail">
+              <ul class="desktop">
+                <li class="icons">
+                  <img src="image/Ellipse-1.png" alt="Javascript logo" class="spac">
+                  <span class="spac"><a href="#">Javascript</a></span>
+                </li>
+                <li class="icons">
+                  <img class="spac" src="image/Ellipse-1-(1).png" alt="HTML logo">
+                  <span class="spac"><a href="#">HTML</a></span>
+                </li>
+                <li class="icons">
+                  <img src="image/Ellipse-1-(2).png" alt="CSS logo" class="spac">
+                  <span class="spac"><a href="#">CSS</a></span>
+                </li>
+              </ul>
+            </div>
+            <div class="language-detail">
+              <ul>
+                <li class="lang icon">
+                  <span class="word">Frameworks</span>
+                  <img src="image/arrow1.png" alt="forward arrow icon">
+                </li>
+                <li class="lang icon">
+                  <span class="word">Skills</span>
+                  <img src="image/arrow1.png" alt="forward arrow icon">
+                </li>    
+               </ul>
+            </div>
+          </div>
```

Comment:
Kindly split the about section into two blocks the right one is `40%` and the left one is `60%`.

![image](https://user-images.githubusercontent.com/24703171/155535477-79bdcbb5-56d7-4d43-95f8-6cd824bfd18c.png)